packages/dynamics-lib/src/queries/__tests__/pocl-validation-error.queries.spec.js

```diff
@@ -0,0 +1,23 @@
+import { findPoclValidationErrors } from '../pocl-validation-error.queries.js'
+
+describe('POCL validation error Queries', () => {
+  describe('findPoclValidationErrors', () => {
+    it('applies default filter if no status is provided', async () => {
+      const query = findPoclValidationErrors()
+      expect(query.toRetrieveRequest()).toEqual({
+        collection: 'defra_poclvalidationerrors',
+        filter: 'statecode eq 0',
+        select: expect.any(Array)
+      })
+    })
+  })
+  it('filters by status and default filter if status provided', async () => {
+    const status = { id: 'test-status-id', label: 'Ready for Processing' }
```

Comment:
Very minor, but these 2 tests don't need to be async

Reply:
Good catch, thanks